sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessageSerializer.java

```diff
@@ -141,7 +150,7 @@ public int getSize(Message amqpMessage) {
         }
 
         if (brokeredMessage.getApplicationProperties() != null) {
-            amqpMessage.setApplicationProperties(new ApplicationProperties(brokeredMessage.getApplicationProperties()));
+            amqpMessage.setApplicationProperties(new ApplicationProperties(convertToDescribedType(brokeredMessage.getApplicationProperties())));
```

Comment:
nit: split this into another line. nested function calls are sometimes hard to understand.